dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/twitter/TwitterTokenServices.java

```diff
@@ -86,7 +86,7 @@ public void getRequestToken(HttpServletRequest request, HttpServletResponse resp
             try {
                 SocialUtil.validateEndpointWithQuery(authzEndpoint);
             } catch (SocialLoginException e) {
-                Tr.error(tc, "FAILED_TO_REDIRECT_TO_AUTHZ_ENDPOINT", new Object[] { config.getUniqueId(), e.getMessage() });
+                Tr.error(tc, "FAILED_TO_REDIRECT_TO_AUTHZ_ENDPOINT", new Object[] { config.getUniqueId(), e });
```

Comment:
These cases - where the exception object is of type `SocialLoginException` - are ones that I'd leave the call to `getMessage()` as it is.